changelogs/8.7.asciidoc

```diff
@@ -0,0 +1,51 @@
+- `transaction.success_count` has been moved to `event.success_count` {pull}9819[9819]
+- Stop indexing transaction metrics to `metrics-apm.internal` {pull}9846[9846]
+- Stop indexing span destination metrics to `metrics-apm.internal` {pull}9926[9926]
+
+[float]
+==== Deprecations
+
+[float]
+==== Bug fixes
+- We no longer unconditionally block shutdown for 5 seconds when tail-based sampling is enabled {pull}9838[9838]
+- Fix a panic caused by accepting new grpc connections during shutdown {pull}10013[10013]
+
+[float]
+==== Intake API Changes
+- Add `request.id` to `span.context.http` {pull}9673[9673]
+- Add `agent.activation_method` metadata field {pull}9850[9850]
+
+[float]
+==== Added
+- Add `event.success_count` to transaction events and transaction metrics {pull}9819[9819] {pull}10165[10165]
+- Set `_doc_count` for service destination metrics {pull}9931[9931]
+- Add `transaction.duration.summary` to transaction metrics {pull}9973[9973]
+- Add `service.language.name` to service transaction metrics and service summary metrics {pull}10111[10111]
+- Add global labels to service destination metrics {pull}10056[10056]
+- Improve APM UI query performance by producing `1m`, `10m` and `60m` aggregation intervals for transaction metrics. Store them into dedicated data streams `metrics-apm.transaction.${interval}` {pull}9846[9846]
+- Improve APM UI query performance by producing `1m`, `10m` and `60m` aggregation intervals for service destination metrics. Store them into dedicated data streams `metrics-apm.service_destination.${interval}` {pull}9926[9926]
+- Improve APM UI query performance by producing `1m`, `10m` and `60m` aggregation intervals for service metrics. Store them into dedicated data streams `metrics-apm.service.${interval}` {pull}9960[9960]
+- Add a service summary metrics aggregator that writes to `metrics-apm.service_summary.${interval}` with `1m`, `10m` and `60m` intervals {pull}10029[10029]
+- Service metrics and service summary aggregations are now always enabled {pull}10060[10060] {pull}10109[10109]
+- Dedicated overflow buckets for transaction, service, service destination and service summary aggregation to limit cardinality {pull}10069[10069] {pull}9856[9856] {pull}10061[10061]
+- Automatically scale `MaxGroups`, `MaxTransactionGroups` and `MaxServices` based on available memory {pull}9856[9856] {pull}10157[10157]
+- Set data streams with non-default rollup intervals as hidden {pull}10070[10070]
+- Rename service metrics to service transaction metrics {pull}10095[10095]
+- Reduce memory usage of transaction and service transaction aggregations {pull}10178[10178]
+- Metrics data streams now use synthetic source {pull}9756[9756]
+- Translate otel log events to error events {pull}10066[10066]
+- Use Elasticsearch fetcher for agent remote configuration where possible {pull}9720[9720] {pull}10088[10088]
```

Comment:
@lahsivjar could you please add a changelog entry for https://github.com/elastic/apm-server/pull/9722?

@kruskall please try to remember to add changelog entries for major changes like this in the future